client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -1,40 +1,96 @@
 import { useTranslate } from 'i18n-calypso';
 import './style.scss';
+import eye from 'calypso/assets/images/marketplace/eye.svg';
+import support from 'calypso/assets/images/marketplace/support.svg';
+import wooLogo from 'calypso/assets/images/marketplace/woo-logo.svg';
 import { formatNumberMetric } from 'calypso/lib/format-number-compact';
+import PluginDetailsSidebarUSP from 'calypso/my-sites/plugins/plugin-details-sidebar-usp';
 
-const PluginDetailsSidebar = ( props ) => {
-	const {
-		plugin: { active_installs, tested },
-	} = props;
-
+const PluginDetailsSidebar = ( {
+	plugin: {
+		active_installs,
+		tested,
+		isMarketplaceProduct = false,
+		demo_url = null,
+		documentation_url = null,
+	},
+} ) => {
 	const translate = useTranslate();
 
-	return (
-		<>
-			<div className="plugin-details-sidebar__plugin-details-title">
-				{ translate( 'Plugin details' ) }
-			</div>
-			<div className="plugin-details-sidebar__plugin-details-content">
-				{ Boolean( active_installs ) && (
-					<div className="plugin-details-sidebar__active-installs">
-						<div className="plugin-details-sidebar__active-installs-text title">
-							{ translate( 'Active installations' ) }
+	if ( ! isMarketplaceProduct ) {
+		return (
+			<>
+				<div className="plugin-details-sidebar__plugin-details-title">
+					{ translate( 'Plugin details' ) }
+				</div>
+				<div className="plugin-details-sidebar__plugin-details-content">
+					{ Boolean( active_installs ) && (
+						<div className="plugin-details-sidebar__active-installs">
+							<div className="plugin-details-sidebar__active-installs-text title">
+								{ translate( 'Active installations' ) }
+							</div>
+							<div className="plugin-details-sidebar__active-installs-value value">
+								{ formatNumberMetric( active_installs, 'en' ) }
+							</div>
 						</div>
-						<div className="plugin-details-sidebar__active-installs-value value">
-							{ formatNumberMetric( active_installs, 'en' ) }
-						</div>
-					</div>
-				) }
-				{ Boolean( tested ) && (
-					<div className="plugin-details-sidebar__tested">
-						<div className="plugin-details-sidebar__tested-text title">
-							{ translate( 'Tested up to' ) }
+					) }
+					{ Boolean( tested ) && (
+						<div className="plugin-details-sidebar__tested">
+							<div className="plugin-details-sidebar__tested-text title">
+								{ translate( 'Tested up to' ) }
+							</div>
+							<div className="plugin-details-sidebar__tested-value value">{ tested }</div>
 						</div>
-						<div className="plugin-details-sidebar__tested-value value">{ tested }</div>
-					</div>
+					) }
+				</div>
+			</>
+		);
+	}
+	const supportLinks = [
+		{ href: 'https://automattic.com/privacy/', label: translate( 'See privacy policy' ) },
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **102** times:
`translate( 'privacy-policy' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugin_details_sidebar_items)